sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -3119,8 +3115,6 @@ atomic_update_impl(T *p, __ESIMD_NS::simd<Toffset, N> offsets,
 /// @anchor usm_atomic_update1
 /// @brief Single-argument variant of the atomic update operation.
 ///
-/// TODO: Document the prototypes similar to other documentation.
```

Comment:
Please keep this comment, it is really convenient to be able to observe all the prototypes in one place

Reply:
Done in https://github.com/intel/llvm/pull/11833/commits/4689604e36d22087fb0a63da4837be828cca8888.